blockchain/My-Wallet-V3-iOS
# feat(KYC | Update Settings Screen)

## Objective

Updates settings with a new visual spec and KYC status cell indicator.

## Description

Added static table cell to Settings.storyboard and added various methods to check the current verification status of the user using `KYCNetworkRequest`. 

## How to Test

Launch the app, go to settings and verify the new visual design matches the spec.  The KYC flow cell does not currently launch from the cell.

## Screenshot
![simulator screen shot - iphone 8 plus - 2018-08-14 at 14 44 57](https://user-images.githubusercontent.com/1316251/44111508-a5341cd2-9fd0-11e8-8199-83e73fd49656.png)


*If the change is UI related, paste a screenshot here.*

## Related Information

* Ticket Number:

## Merge Checklist
